examples/custom-resources/custom-listeners/README.md

```diff
@@ -0,0 +1,156 @@
+# Custom HTTP Listeners
+
+In this example, we will configure a VirtualServer resource with custom HTTP listeners.
+This will allow HTTP and/or HTTPs based requests to be made on non-default ports.
+
+## Prerequisites
+
+1. Follow the [installation](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/)
+   instructions to deploy the Ingress Controller with custom resources enabled.
+2. Ensure the Ingress Controller is configured with the `-global-configuration` argument:
```

Comment:
should probably specify that lb/nodeport service has to configured with extra ports

Reply:
Good point yeah. Nice catch!